tests/unit/admin/test-class-wc-rest-payments-settings-controller.php

```diff
@@ -130,19 +124,18 @@ public function set_up() {
 				->will( $this->returnValue( false ) );
 
 			$mock_payment_methods[ $mock_payment_method->get_id() ] = $mock_payment_method;
-		}
 
-		$this->upe_gateway    = new UPE_Payment_Gateway(
-			$this->mock_api_client,
-			$this->mock_wcpay_account,
-			$customer_service,
-			$token_service,
-			$action_scheduler_service,
-			$mock_payment_methods,
-			$mock_rate_limiter,
-			$order_service
-		);
-		$this->upe_controller = new WC_REST_Payments_Settings_Controller( $this->mock_api_client, $this->upe_gateway );
+			$this->mock_payment_gateways[ $mock_payment_method->get_id() ] = new UPE_Payment_Gateway(
+				$this->mock_api_client,
+				$this->mock_wcpay_account,
+				$customer_service,
+				$token_service,
+				$action_scheduler_service,
+				$mock_payment_method,
```

Comment:
For the test cases in this file, I changed how we test UPE gateways. Instead of testing 1 UPE gateway instance, it now tests individual UPE gateway objects injected with the individual UPE payment method.